Tests/FunctionalTest.php

```diff
@@ -85,6 +86,27 @@ public function testAbstractClassNotRegistered()
         $this->assertFalse(static::$kernel->getContainer()->has('dunglas\actionBundle\tests\fixtures\testbundle\action\abstractaction'));
     }
 
+    public function testAnonymousClassNotRegistered()
+    {
+        if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
+            $hasFailed = null;
+            try {
+                static::bootKernel();
+                $this->assertTrue(static::$kernel->getContainer()->has('dunglas\actionBundle\tests\fixtures\anonymousaction\ananonymousaction'));
+                /** @var AnAnonymousAction $action */
+                $action = static::$kernel->getContainer()->get('dunglas\actionBundle\tests\fixtures\anonymousaction\ananonymousaction');
+                $this->assertEquals('Ho hi', $action());
+                $hasFailed = false;
+            } catch (\Symfony\Component\DependencyInjection\Exception\RuntimeException $e) {
```

Comment:
You mean, just let them happen (without try/catch) ?
Didn't find an "assert exception not thrown" in PHPUnit...